backend/app/services/chat/config/model_resolver.py

```
@@ -764,6 +797,124 @@ def _find_model_with_namespace(
+            return None, runtime_model_spec
+        return None, None
+
+    if normalized_type == "public":
+        model = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == 0,
+                Kind.kind == "Model",
+                Kind.name == model_name,
+                Kind.namespace == normalized_namespace,
+                Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)